cirq-core/cirq/interop/quirk/cells/parse.py

```diff
@@ -191,7 +191,9 @@ def feed_op(could_be_binary: bool, token: Any) -> None:
             elif token.unary_action is not None:
```

Comment:
consider using a walrus operator here:
```suggestion
            elif (action := token.unary_action) is not None:
```
Then `lambda _, b: action(b)` should work.

Reply:
Ack.  I feel the current version makes it a bit more clear why is the attribute stored in a local var.